components/Statistics/StatisticsLocationsReport.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { LoadSpinner } from '../Loading'
+import { PieChart, Pie, Tooltip, Legend, Cell, ResponsiveContainer } from 'recharts'
+import styled from 'styled-components'
+import { useStatisticsAPI } from '../../lib/statistics/statisticsHooks'
+
+const COLORS = ['#0088FE', '#00C49F', '#FFBB28', '#FF8042', '#653CAD']
+
+const Container = styled.div`
```

Comment:
this and the previous module have 80% the same code - only the data obtained and the display widget change. This implies some factoring is possible.  i.e. a wrapper page or a curried function that takes the data type and widget to use, or a tab component that takes the data source as a prop and the display component as a child.